frontend/src/MessageBoard/components/MessageBoard.js

```diff
@@ -0,0 +1,166 @@
+	margin-right: 8%;
+	top: 20px;
+	right: 18%;
+	position: absolute;
+`;
+
+const StyledLink = styled(Link)`
+	color: black;
+	text-decoration: none;
+	margin: 5px;
+	font-weight: bold;
+`;
+
+const TeamName = styled.div`
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin-top: 40px;
+`;
+
+const Teamlogo = styled.div`
+	display: flex;
+	align-items: center;
+
+	& button {
+		height: 50%;
+		margin-left: 5px;
+	}
+`;
+
+const Logo = styled.img`
+	display: inline;
+	margin-right: 5px;
+	border-radius: 45px;
+`;
+
+const MessagesContainer = styled.div`
+	margin: 30px 20px;
+	border: 1px solid black;
+`;
+
+class MessageBoard extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			teamName: 'TeamHome',
+			user: 'some id',
+			messages: [
+				{
+					_id: '5c358baf93b69c7c387fd843',
+					user: { firstName: 'Joe', lastName: 'Bob', avatar: null },
+					team: 'Team Home',
+					title: 'Example',
+					content:
+						'This is purely an example, none of this is from the database...yet. The image is hosted',
+					images: [
+						'https://res.cloudinary.com/massamb/image/upload/v1547153046/956_IMG_3495.jpg'
+					],
+					tags: ['test', 'tags'],
+					comments: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					],
+					subscribedUsers: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					]
+				},
+				{
+					_id: '5c358baf93b69c7c387fb843',
+					user: { firstName: 'Joe', lastName: 'Bob', avatar: null },
+					team: 'Team Home',
+					title: 'Example',
+					content:
+						'This is purely an example, none of this is from the database...yet. The image is hosted',
+					images: [
+						'https://res.cloudinary.com/massamb/image/upload/v1547153046/956_IMG_3495.jpg'
+					],
+					tags: ['test', 'tags'],
+					comments: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					],
+					subscribedUsers: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					]
+				},
+				{
+					_id: '5c358baf93b69c7c387fc843',
+					user: { firstName: 'Joe', lastName: 'Bob', avatar: null },
+					team: 'Team Home',
+					title: 'Example',
+					content:
+						'This is purely an example, none of this is from the database...yet. The image is hosted',
+					images: [
+						'https://res.cloudinary.com/massamb/image/upload/v1547153046/956_IMG_3495.jpg'
+					],
+					tags: ['test', 'tags'],
+					comments: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					],
+					subscribedUsers: [
+						'5c358baf93b69c7c387fb843',
+						'5c358baf93b69c7c387fb234',
+						'5c358baf93b69c7c387fb234'
+					]
+				}
+			],
+			isAdmin: true
+		};
+	}
+
+	componentDidMount() {
+		// TODO: Get user and team info from database and store it in state
```

Comment:
Y'all have Redux going, and shouldn't be planning on storing large swatches of data like this in local state. 👎